tests/browser/contextual-ee.browser.js

```diff
@@ -65,3 +65,40 @@ test('ee.abort condition not met', function (t) {
 
   t.end()
 })
+
+test('EE clears eventBuffer (ee.backlog) after abort', function (t) {
+  ee.aborted = false
+  if (ee.backlog) {
+    delete ee.backlog.feature
+  }
+
+  ee.buffer(['eventType'], 'feature')
+
+  t.equal(ee.backlog.feature instanceof Array, true, 'feature backlog is an array')
+  t.equal(ee.backlog.feature.length, 0, 'feature backlog is an empty array')
+
+  ee.emit('eventType', [], {})
+
+  t.equal(ee.backlog.feature.length, 1, 'emitted event was buffered')
+
+  ee.abort()
+
+  t.equal(Object.keys(ee.backlog).length, 0, 'EE backlog was cleared after abort')
+
+  t.end()
+})
+
+test('EE does not buffer events after it has been aborted', function (t) {
```

Comment:
This test is passing even without the change in the emitter since `ee.abort()` resets the event backlog. Under what circumstances did the buffer get re-initialized?

Reply:
This test didn't take into account the `handle` behavior of calling buffer & emit after abort. I've updated the test to include that behavior and changed where we check the event emitter has been aborted to try and make things clearer.